pyproject.toml

```diff
@@ -63,6 +63,7 @@ deepdiff = ">=7.0.1"
 scikit-image = {version = "^0.23.2", optional = true}
 pandas = {version = "^2.2.2", optional = true}
 pytest-mock = {version = "^3.14.0", optional = true}
+accelerate = {version = "^0.32.1", optional = true}
```

Comment:
@aliberts wasn't there a thing before about changing `^` to `>=` to avoid pinning major version? Also looks like the 4 dependencies above have the `^` as well.

Reply:
Ah yes, by default poetry adds dependencies with `^` but we actually want `>=`.
I should probably find a way to test that in CI (although there are a few dependencies where we actually want `^` I think, the big ones like torch for example).
Okay to me for this merge, I'll take care of them in a future PR. Adding it to the backlog